package com.revature.mockERS.exceptions;

public class ResourceNotFoundException extends RuntimeException {
    //todo Exception found in Revabooks
}
